### PR TITLE
Add documentation details for PostgreSQL

### PR DESCRIPTION
**Pull request by vfiftyfive**
### Description

Add documentation details for PostgreSQL with feedback from ufirst.

### CODE REVIEW

File: docs/features/postgresql/index.mdx [hunk 1]
```diff
@@ -21,23 +21,42 @@ export const postgres_tutorials = [
...
-In addition, Otterize can map the access to your PostgreSQL database, showing you which service is accessing which database, table and which operation it's performing. This can be used to automatically generate the `ClientIntents` declarations.
+Otterize enables secure and efficient access to your PostgreSQL database within your Kubernetes environment. By utilizing [Intent-Based Access Control](/overview/intent-based-access-control) (IBAC), Otterize creates just-in-time username-and-password pairs for your services, delivering them as Kubernetes Secrets that can be mounted to files or mapped to environment variables, as well as `GRANT`ing access to databases and tables based on these declarations. Additionally, Otterize can automatically generate IBAC declarations (`ClientIntents`) by analyzing your database access patterns, determining which services are accessing which databases, tables, and the specific operations they're performing.
```
Comment (evyatarmeged) [inline docs/features/postgresql/index.mdx:24]: Maybe rephrase "within your Kubernetes environment" so it will be clear we can configure DBs outside of the cluster as well.

**Review by evyatarmeged — APPROVED** [review]

File: docs/features/postgresql/index.mdx (hunk 1 above)
Comment (vfiftyfive) [reply docs/features/postgresql/index.mdx]: done
